### PR TITLE
Feat/sudoku cells

### PR DESCRIPTION
**Pull request by patrikglanemann**
Moving from codesandbox to the actual project and slowly creating small branches.

What I did:
- Create component for individual Sudoku cells
- Add Function to create and render cells. (only three for now, not the whole grid)

Also I included eslint in my project and it throws me two errors
- React...missing in props validation
- React must be in scope when using JSX react/react-in-jsx-scope
(I imported React. Typical mistake seems to be  to spell React in lowercase, but I didn't)
That is why I set some rules in eslintrc.json to ignore the errors for now.

### CODE REVIEW

**Review by adrianammt — COMMENTED** [review]
Looks good to me!

File: .eslintrc.json [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-    ],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": 12
-    },
-    "plugins": [
-        "react"
-    ],
```
Comment (namirsab) [inline .eslintrc.json:19]: You don't need this eslint config actually. Create React App comes already with eslint configured (check your package.json) and shouldn't give any issues for the issues you mentioned

**Review by namirsab — CHANGES_REQUESTED** [review]
Hey @patrikglanemann 
One little step for a man, one big step for you!
First PR! just a comment regarding eslint!

File: .eslintrc.json (hunk 1 above)
Comment (patrikglanemann) [reply .eslintrc.json:19]: Ah I wasn't aware that Create React App already includes eslint. Thanks for the info:)

**Pull request merged**